src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -93,6 +95,54 @@ public void given_initiate_tasks_with_follow_up_overdue_category_then_cancel_tas
         taskToTearDown = taskCreatedAsResultOfTheMultipleDmnRule;
     }
 
+    @Test
+    public void given_initiated_tasks_with_delayTimer_toCurrentTime_then_cancel_task() throws InterruptedException {
+        // create task1
+        String caseIdForTask1 = UUID.randomUUID().toString();
+        String taskIdDmnColumn = "followUpOverdueRespondentEvidence";
+        String task1Id = initiateTaskForGivenId(caseIdForTask1, "requestRespondentEvidence",
+                                                "", "awaitingRespondentEvidence",
+                                                false, taskIdDmnColumn);
+
+        // create task2
+        String caseIdForTask2 = UUID.randomUUID().toString();
+        final String task2Id = initiateTaskForGivenId(caseIdForTask2, "uploadHomeOfficeBundle",
+                                                "", "awaitingRespondentEvidence",
+                                                false, "reviewRespondentEvidence");
+
+        // Then cancel the task1
+        sendMessage(caseIdForTask1, "uploadHomeOfficeBundle", "awaitingRespondentEvidence", "", false);
+        assertTaskDoesNotExist(caseIdForTask1, taskIdDmnColumn);
+        assertTaskDeleteReason(task1Id, "deleted");
+
+        // add tasks to tear down.
+        taskToTearDown = task2Id;
+    }
+
+    @Test
+    public void given_initiated_tasks_with_delayTimer_toFuture_then_cancel_task() {
```

Comment:
We should only cancel a task with delayTimer_toFuture if there is a Follow-up overdue event. Should the test be adjusted to quality this behaviour?